protobuf/transform_test.go

```diff
@@ -0,0 +1,282 @@
+package protobuf
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/src-d/proteus/resolver"
+	"github.com/src-d/proteus/scanner"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestToLowerSnakeCase(t *testing.T) {
+	cases := []struct {
+		input    string
+		expected string
+	}{
+		{"fooBarBaz", "foo_bar_baz"},
+		{"FooBarBaz", "foo_bar_baz"},
+		{"foo1barBaz", "foo1bar_baz"},
+	}
+
+	for _, c := range cases {
+		require.Equal(t, c.expected, toLowerSnakeCase(c.input))
+	}
+}
+
+func TestToUpperSnakeCase(t *testing.T) {
```

Comment:
The same here: It will fail with fooBAR for example. (result:FOO_B_A_R. I think the expected result should be: FOO_BAR)